### ISSUE TITLE
508-defect-1 [SCREEN READER, PDF]: 10-10ez Spanish pdf must have Spanish as its selected primary language 

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22651]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh Kim

## User Story or Problem Statement

As a Spanish screen reader user, I want to be able to read Spanish pdf forms in a native accent so that I can complete them.

## Details

The Spanish version of the 10-10ez pdf is missing a primary language tag. Without a primary language tag, Spanish content may be read out in an English accent which results in an incomprehensible document. This will also fail to meet Section 508 requirements as defined in [WCAG 2.0 PDF19, language in page](https://www.w3.org/TR/WCAG20-TECHS/PDF19.html).

From a human lens, this may mean the 10-10ez will be completely inaccessible to blind or low vision Spanish speaking Veterans/family members. 

## Acceptance Criteria

- [ ] A primary language of Spanish is selected for the pdf

## Environment

* 10-10ez Spanish pdf

## Steps to Recreate

1. Open the 10-10ez Spanish pdf using Adobe Acrobat
2. Click on the tools tab and add Accessibility
3. Open the Accessibility tab and click "Accessibility Check" -> Start checking
4. Confirm primary language fails

## Proposed Solution (if known)

This can be fixed in under a minute. In Adobe Acrobat, click on File, then, Properties, and then click on the Advanced tab. There is a language dropdown under reading options that can be set to Spanish.

## WCAG or Vendor Guidance (optional)

[WCAG 2.0 PDF19, language in page](https://www.w3.org/TR/WCAG20-TECHS/PDF19.html)

## Screenshots or Trace Logs

<img width="1434" alt="Screen Shot 2021-04-05 at 1 42 23 PM" src="https://user-images.githubusercontent.com/14154792/113615633-19276a00-9622-11eb-9887-53989ed9ca2b.png">

<img width="1145" alt="Screen Shot 2021-04-05 at 2 00 23 PM" src="https://user-images.githubusercontent.com/14154792/113615592-0e6cd500-9622-11eb-9e5f-fd964b5a3888.png">

**Comment by DanielleThierryUSDSVA** [comment]
Good morning, @joshkimux. Just noting that I've connected with the VHA forms program office about this and will send an update when it's fixed. Thanks!

**Comment by joshkimux** [comment]
That's awesome!!! Thank you so much ❤️

**Comment by DanielleThierryUSDSVA** [comment]
@joshkimux Quick update. Forms team is connecting with 508 office on this to fix.

**Comment by joshkimux** [comment]
@DanielleThierryUSDSVA did the 508 office ever provide a response to this?
